internal/sail/client/client.go

```diff
@@ -10,20 +10,23 @@ import (
 	"github.com/windmilleng/tilt/internal/hud/webview"
 	"github.com/windmilleng/tilt/internal/logger"
 	"github.com/windmilleng/tilt/internal/model"
+	sailCommon "github.com/windmilleng/tilt/internal/sail/common"
```

Comment:
this seemed clearer than just referring to it as "common" in the code (and more go-like than naming the package `sail_common`), but idk if this stylistic 🤷‍♀️